src/test/java/core/basesyntax/handler/ReturnOperationHandlerImplTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.handler;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.FruitsStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.OperationHandler;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReturnOperationHandlerImplTest {
+    private static FruitTransaction fruitTransaction;
+    private static OperationHandler operationHandler;
+
+    @BeforeAll
+    static void setUpBeforeAll() {
+        operationHandler = new ReturnOperationHandlerImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        fruitTransaction = new FruitTransaction.FruitBuilder()
+                .setOperationType(Operation.RETURN)
+                .setFruitName("apple")
+                .setFruitQuantity(100)
+                .build();
+    }
+
+    @Test
+    void applyOperation_ReturnFruitsToEmptyStorage_AllValidData_ok() {
```

Comment:
name for the method should start from lower case letter after the underscore 
![image](https://github.com/mate-academy/jv-fruit-shop-tests/assets/32339836/71f8183d-fce4-4252-9e0d-332d90005607)
Fix it for all test names